doc/rtd/topics/datasources.rst

```diff
@@ -30,12 +30,12 @@ The following is a list of documents for each supported datasource:
    datasources/aliyun.rst
    datasources/altcloud.rst
    datasources/azure.rst
-   datasources/ec2.rst
-   datasources/e24cloud.rst
    datasources/cloudsigma.rst
    datasources/cloudstack.rst
    datasources/configdrive.rst
    datasources/digitalocean.rst
+   datasources/ec2.rst
+   datasources/e24cloud.rst
```

Comment:
Sorted alphabetically.

Reply:
These two, specifically, are still in reverse alphabetical order, I think.

Reply:
It depends whether the numbers come before or after the letters. The alphabet does not define it, so I will drop that change.

Reply:
For clarity, this is my basis for saying they are in reverse order:

```shell
$ sort << EOF
>    datasources/ec2.rst
>    datasources/e24cloud.rst
> EOF
   datasources/e24cloud.rst
   datasources/ec2.rst
```

(Or do you get a different order with a different locale?)